app/uk/gov/hmrc/eoricommoncomponent/frontend/util/Require.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eoricommoncomponent.frontend.util
+
+object Require {
+
+  @inline final def requireThatUrlValue(requirement: Boolean, message: => Any) {
+    if (!requirement)
+      throw InvalidUrlValueException("invalid value: " + message)
+  }
+
+}
```

Comment:
Probably this method can be tested separately too

Reply:
Test added.  Could you take a look please?